common/model-views.cpp

```diff
@@ -3837,10 +3837,13 @@ namespace rs2
                 configurations::update::allow_rc_firmware,
                 false );
             bool is_rc = ( product_line == RS2_PRODUCT_LINE_D400 ) && allow_rc_firmware;
-            std::string available_fw_ver = get_available_firmware_version( product_line );
+            std::string PID = product_line == RS2_PRODUCT_LINE_L500 ? dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID) : "";
```

Comment:
Why only for L500? We should get it for all cameras

Reply:
(just be careful on line 3851 (!!) -- need to check with and then without PID)

Reply:
create_default_fw_table creates the map with PID only for L500 and for all the rest skus with empty string and that's how we use the table after that, otherwise I need to create the map with PID for all the sku, do you think its better ?

Reply:
done